packages/server/migrations/201908021430000-channel-membership-type.js

```diff
@@ -0,0 +1,39 @@
+const {
+  CHANNEL_MEMBERSHIP_TYPE,
+} = require('../src/models/enums/channel-membership-type');
+
+module.exports = {
+  up: async function (queryInterface, Sequelize) {
+    try {
+      await queryInterface.sequelize.query(`BEGIN`);
+
+      await queryInterface.addColumn('ChannelMemberships', 'type', {
+        type: Sequelize.ENUM([
+          CHANNEL_MEMBERSHIP_TYPE.ADDED,
+          CHANNEL_MEMBERSHIP_TYPE.JOINED,
+          CHANNEL_MEMBERSHIP_TYPE.INVITED,
```

Comment:
These aren't obvious to me. Does invited mean via email? Maybe something more like,
```
ADDED_IN_APP_BY_ADMIN
ADDED_BY_EMAIL_BY_ADMIN
USER_JOINED_DIRECTLY
```
or similar? I'm fine with slightly longer names for the sake of readability.

Reply:
Thanks, updated.